portal/templates/redesign/play_new.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'redesign/play_new/base_new.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{block.super}}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block content %}
+{{block.super}}
+
+<script type="text/javascript" src='{% static "portal/js/lib/jquery.js" %}'></script>
+<script type="text/javascript" src='{% static "portal/js/lib/jquery.colorbox.js" %}'></script>
+
+<script type="text/javascript">
+    $(document).ready(function(){
+        //Examples of how to assign the Colorbox event to elements
```

Comment:
That comment probably shouldn't still be here.  And if it's correct, you should probably take away any "example" code!